vignettes/openxlsx2_style_manual.Rmd

```diff
@@ -112,7 +116,7 @@ new_cellxfs <- create_cell_style(
 wb$styles_mgr$add(new_cellxfs, "new_styles")
 
 # assign the new cell style to the header row of our data set
-cell <- sprintf("A1:%s1", int2col(nrow(mat)))
+cell <- wb_dims(col = seq_len(ncol(mat)), row = 1)
```

Comment:
Ah yes, this is how I calculate the dimensions. Especially useful if one wants to create complicated worksheets with formulas all over the place (I do not really believe that `wb_dims()` would be very useful, but I have to try this sometime soon)

Reply:
I think it could be worth showing the 2 approaches then ?